### DISCUSSION TITLE
Summarized notifications per email or on screen

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#13510] [Archive]
Hi!

What would be the best way to send a digest (email) notification of changes in a collection?

Let's say we want to notify users every morning of the new items in the "products" collection, that have just been created the day before. Or we want to notify users of all products that had their status changed to "End-Of-Life" the day before?

Any thoughts would be greatly appreciated!

Regards,

Kees.

**Comment by rijkvanzanten** [comment]
I would probably use a schedule hook that runs daily to query directus_activity to find all the changes in the last 24 hours (filtered down to `create`/`update` where collection = `products`) then use the MailService to send a summary to the configured email addresses? 🤔

**Reply by keesvanbemmel** [reply]
Sounds like a plan @rijkvanzanten! How do we get itemsservice updates to show up in that list? Do we need to add accountability for that?

**Reply by rijkvanzanten** [reply]
By default, accountability is enabled to track both the event and the payload that was submitted in updates, so you could query directus_revisions to get the changes that we're made efficiently!

**Reply by keesvanbemmel** [reply]
Sorry, my question wasn't clear @rijkvanzanten. We're using the ItemsService updateOne quite regularly without accountability set in the function call, because at that stage we don't need it (admin rights is good enough).

These changes aren't in the activity/revisions list. Is that because accountability isn't set? Or do we need to set anything else in the updateOne call?

**Reply by keesvanbemmel** [reply]
The answer to my own question after a test: yes, you need to set accountability on the itemsservice for revisions and activity to be tracked (if enabled of course)
